src/utils/display-text/format.ts

```diff
@@ -109,5 +109,13 @@ export function calculateProfit(
   return isLong ? profitLong : profitShort;
 }
 
-export const stringToFixedPrecision = (value: string, precision: number) =>
-  parseFloat(value).toFixed(precision);
+export const countDecimals = value => {
+  if (Math.floor(value) === value || !value.split('.')[1]) return 0;
+  return value.split('.')[1].length || 0;
```

Comment:
The OR part is unnecessary here since you do all the checks before + you can use newly created variable as `decimalPart.length`.